opencti-platform/opencti-front/src/components/ItemConfidence.tsx

```diff
@@ -35,11 +34,21 @@ const ItemConfidence: FunctionComponent<ItemConfidenceProps> = ({ confidence, va
   const { level: confidenceLevel } = useLevel(entityType, 'confidence', confidence);
 
   const style = variant === 'inList' ? classes.chipInList : classes.chip;
+  const multilineChip = {
```

Comment:
It's to be able to display the chip in multiline if the text is long. It is currently cut, and with the new default confidence scale, it would have been always cut like this :
![Capture d'écran 2023-07-21 100651](https://github.com/OpenCTI-Platform/opencti/assets/668192/718e3ca1-12fe-428a-9e50-48e99d84a3bd)